easybuild/easyconfigs/t/texlive/texlive-20230313-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,64 @@
+# Based off the 2017 version by John Dey jfdey@fredhutch.org
+# https://github.com/easybuilders/easybuild-easyconfigs/pull/5085
+easyblock = 'Tarball'
+
+name = 'texlive'
+version = '20230313'
+local_version_year = version[:4]
+
+homepage = 'https://tug.org'
+description = """TeX is a typesetting language. Instead of visually formatting your text, you enter your manuscript
+ text intertwined with TeX commands in a plain text file. You then run TeX to produce formatted output, such as a
+ PDF file. Thus, in contrast to standard word processors, your document is a separate file that does not pretend to
+ be a representation of the final typeset output, and so can be easily edited and manipulated."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = [
+    'ftp://tug.org/texlive/historic/{}/'.format(local_version_year),
```

Comment:
Is `.format()` acceptable, or do I need to use the `%` formatting syntax that I can never remember?

Reply:
It would usually be `%(local_version_year)s` but in other cases something like:
`sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]`

Reply:
Possibly, hence the second suggestion.